File: comms/bulk-messaging/senders/search-senders/search-senders.go
package example

import (
    context "context"

    twiliocomms "github.com/twilio/twilio-comms-go/twilio"
    client "github.com/twilio/twilio-comms-go/twilio/client"
    option "github.com/twilio/twilio-comms-go/twilio/option"
)

func do() {
    client := client.NewWithOptions(
        option.WithBasicAuth(
            "<username>",
            "<password>",
        ),
    )
    request := &twiliocomms.SendersSearchRequest{
        Address: "+14153902337",
        Channel: twiliocomms.SenderCommunicationChannelSms,
    }
    client.Senders.Search(
        context.TODO(),
        request,
    )
}

File: comms/bulk-messaging/senders/search-senders/search-senders.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "<username>",
        authToken: "<password>",
    });
    await client.senders.search({
        address: "+14153902337",
        channel: "SMS",
    });
}
main();
